tools/add_triage_signature.py

```diff
@@ -503,6 +505,29 @@ class CNIConfigurationError(Signature):
     def __init__(self, jira_client):
         super().__init__(jira_client, comment_identifying_string="h1. No CNI configuration")
 
+    @staticmethod
+    def _get_host_remote_hosts_ip_addresses(host):
+        """
+        Given a host object, return a list of all IP addresses of the remote hosts in this host's
+        connectivity stanza.
+        """
+        try:
+            return [(remote_host['host_id'], remote_host['l3_connectivity'][0]['remote_ip_address'])
+                    for remote_host in
+                    json.loads(host["connectivity"])['remote_hosts']]
+        except Exception:
```

Comment:
in which case we might have this exception?

Reply:
KeyError mostly, malformed service output :man_shrugging: